### PR TITLE
use correct singular or plural sentences for pil warnings

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

**Comment by lennym** [comment]
This is an absolute shit-show of embedded conditions and is tantamount to abuse of mustache.

Could we just just build the strings in code instead? It's single-use content so I don't really see any benefit in pulling it out into a separate location and the way it is here is basically impossible to read.

File: pages/dashboard/views/index.jsx [hunk 1]
```diff
@@ -47,6 +48,29 @@ const EstablishmentPanel = ({ establishment, profile }) => {
   );
 };
 
+const getWarningText = review => {
+  let warningText = `${review.name} has `;
+
```
Comment (lennym) [inline pages/dashboard/views/index.jsx:53]:
```suggestion
  const plural = (item, count) `${item}${count === 1 ? '' : 's'}`;
```

And then scatter liberally below.

Comment (wheelsandcogs) [reply pages/dashboard/views/index.jsx:53]: we still need to handle `is/are` `it's/their` so it's probably simpler just to have separate sentences.

Comment (lennym) [reply pages/dashboard/views/index.jsx:53]: Legit. Words are a pain.

**Pull request merged**